app/components/navigation_bar.html.erb

```diff
@@ -0,0 +1,22 @@
+<div class="app-primary-navigation">
+    <div class="app-primary-navigation govuk-!-display-none-print">
+    <div class="govuk-width-container app-primary-navigation--justify-between">
+        <nav class="app-primary-navigation__nav" aria-label="main menu">
+        <ul class="app-primary-navigation__list">
+            <li class="app-primary-navigation__item">
+                <a class="app-primary-navigation__link" href="#">Claims</a>
```

Comment:
I believe we agreed to use localisation for strings, so this should be
```suggestion
                <a class="app-primary-navigation__link" href="#"><%= t("claims") %></a>
```

or similar.

Also, if this is meant to be a claims-specific component, it should before be named `claims_navigation_bar.html.erb`. If it's meant to be generic, it should probably take the primary nav title as an argument.